### DISCUSSION TITLE
Controlled sources

### DISCUSSION THREAD

**Discussion by cmat21** [ra3xdh/qucs_s#390] [Ideas]
Hi, I noticed that all the controlled sources have the same symbol style and when you zoom out the difference between them is not clear. Also, all controlled sources have the same name "SRC1", you could use a name directly related to the source type, such as "VCVS1". Additionally, all controlled sources have the same letter "G" to describe the foward transfer factor.
Could you consider changing it?
![voltages_sources](https://github.com/ra3xdh/qucs_s/assets/70080894/69dcd7e1-75ee-4364-81c5-64a30577089b)

**Comment by ra3xdh** [comment]
> that all the controlled sources have the same symbol style and when you zoom out the difference

In principle the device symbol may be changed, but the pins location must be preserved. Otherwise old schematics will not load. But keep in mind that sources symbols exists in Qucs almost 20 years. I would not to change the things that works and not broken. 

>  the same name "SRC1"

The default name is not related to the SPICE letter. But in principle it could be changed. It's need to check if the device name affects Qucsator simulation. 

> all controlled sources have the same letter "G" to describe the forward transfer factor

This cannot be changed, because Qucsator netlist requires to set the parameter name. If the parameter name is changed, the Qucsator simulation will be broken. SPICE netlist entry is formed by the special procedure and not related to parameter names. Qucsator compatibility must be preserved. See #370
